torchsparse/utils/kernel_region.py

```diff
@@ -1,18 +1,32 @@
 import numpy as np
```

Comment:
We can rename this as `kernel.py` and move `KernelMapKey` here.

Reply:
Fixed.